pkg/workspace/plugins.go

```diff
@@ -40,6 +41,10 @@ const (
 	windowsGOOS = "windows"
 )
 
+var (
+	enableLegacyPluginBehavior = os.Getenv("PULUMI_ENABLE_LEGACY_PLUGIN_SEARCH") != ""
```

Comment:
We have `cmdutil.IsTruthy` which we use other places, and has a slightly nicer semantics.

Reply:
Unfortunately this causes an import cycle:

```
package github.com/pulumi/pulumi/sdk/nodejs/cmd/pulumi-language-nodejs
        imports github.com/pulumi/pulumi/pkg/util/cmdutil
        imports github.com/pulumi/pulumi/pkg/diag
        imports github.com/pulumi/pulumi/pkg/resource
        imports github.com/pulumi/pulumi/pkg/workspace
        imports github.com/pulumi/pulumi/pkg/util/cmdutil
```

Reply:
Sadness.

Reply:
If we really want to, we could break the cycle and pass `enableLegacyPluginBehavior` in as an option to the relevant method. Your call, @swgillespie

Reply:
I think that's cleaner - I'll try that and see how it goes.

Reply:
This didn't go super well, especially since I'm also making changes to other functions in this file that respect the legacy flag. I think it's definitely easiest to leave it as an environment variable here without `cmdutil.IsTruthy`, unless anyone has any objections.

Reply:
No problems here.